Documentation/deploy-workers.md

```diff
@@ -45,11 +44,11 @@ After=flanneld.service
 
 #### Create the kubelet Unit
 
-Create `/etc/systemd/system/kubelet.service` and replace: 
+Create `/etc/systemd/system/kubelet.service` and replace the following variables: 
 
-* Replace `${MASTER_IP}`
-* Replace `${ADVERTISE_IP}` with this nodes publicly routable IP.
-* Replace `${DNS_SERVICE_IP}`
+* `${MASTER_IP}`
+* `${ADVERTISE_IP}`, with this node's publicly routable IP.
+* `${DNS_SERVICE_IP}`
```

Comment:
Can we keep the replace since we use this same syntax in ~10 other places?

```
$ grep -r "* Replace" kubernetes/docs/1.0.3/
kubernetes/docs/1.0.3//configure-kubectl.md:* Replace `${MASTER_IP}`
kubernetes/docs/1.0.3//configure-kubectl.md:* Replace `${CA_CERT}` with the path to the `ca.pem` created in previous steps
kubernetes/docs/1.0.3//configure-kubectl.md:* Replace `${ADMIN_KEY}` with the path to the `admin-key.pem` created in previous steps
kubernetes/docs/1.0.3//configure-kubectl.md:* Replace `${ADMIN_CERT}` with the path to the `admin.pem` created in previous steps
kubernetes/docs/1.0.3//deploy-addons.md:* Replace `${DNS_SERVICE_IP}`
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `${ADVERTISE_IP}` with this machine's publicly routable IP.
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `${ETCD_ENDPOINTS}`
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `${ADVERTISE_IP}` with this nodes publicly routable IP.
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `${DNS_SERVICE_IP}`
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `${ETCD_ENDPOINTS}`
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `${SERVICE_IP_RANGE}`
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `${ADVERTISE_IP}` with this nodes publicly routable IP.
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `$POD_NETWORK`
kubernetes/docs/1.0.3//deploy-master-single.md:* Replace `$ETCD_SERVER` with one host from `$ETCD_ENDPOINTS`
kubernetes/docs/1.0.3//deploy-workers.md:* Replace `${ADVERTISE_IP}` with this nodes publicly routable IP.
kubernetes/docs/1.0.3//deploy-workers.md:* Replace `${ETCD_ENDPOINTS}`
kubernetes/docs/1.0.3//deploy-workers.md:* Replace `${MASTER_IP}`
kubernetes/docs/1.0.3//deploy-workers.md:* Replace `${ADVERTISE_IP}` with this nodes publicly routable IP.
kubernetes/docs/1.0.3//deploy-workers.md:* Replace `${DNS_SERVICE_IP}`
kubernetes/docs/1.0.3//deploy-workers.md:* Replace `${MASTER_IP}`
kubernetes/docs/1.0.3//openssl.md:* Replace `${K8S_SERVICE_IP}`
kubernetes/docs/1.0.3//openssl.md:* Replace `${MASTER_IP}`
```